proto/workload.proto

```diff
@@ -74,8 +74,22 @@ message Service {
   repeated string subject_alt_names = 6;
 }
 
+// Workload represents a workload - an endpoint (or collection behind a hostname).
+// The xds primary key is "uid" as defined on the workload below.
+// Secondary (alias) keys are the unique `network/IP` pairs that the workload can be reached at.
 message Workload {
   // UID represents a globally unique opaque identifier for this workload.
+  // For k8s resources, it is recommended to use the more readable format:
+  //
+  // cluster/group/kind/namespace/name/section-name
+  //
+  // As an example, a ServiceEntry with two WorkloadEntries inlined could become
+  // two Workloads with the following UIDs:
+  // - cluster1/networking.istio.io/v1alpha3/ServiceEntry/default/external-svc/endpoint1
```

Comment:
How about pod?  It is like `cluster1/service` or `cluster1/pod`. Seems not very clear

Reply:
the unit tests in this PR highlight the _recommended_ pod format, although if you wanted to use the pod k8s UID you could also use that

the recommended format for pod would be `cluster1//Pod/default/my-pod`